src/main.cpp

```diff
@@ -12,10 +13,15 @@ int main(int argc, char** argv, char** envp) {
 
         else if (arg == "--quiet" || arg == "-q")
             Debug::quiet = true;
+
+        else if (arg == "--config" || arg == "-c") {
+            configPath = argv[i + 1];
```

Comment:
nit: you can do ++i here and omit the line below with i++

Reply:
Done.